Core/TcpServer.cs

```diff
@@ -139,11 +139,11 @@ public void Start()
                                     break;
 
                                 case OutputCodes.IdleEnter:
-                                    Console.Title = "[IDLE] " + LocalAdmin.BaseWindowTitle;
+                                    LocalAdmin.SetTerminalTitle("[IDLE] " + LocalAdmin.BaseWindowTitle);
```

Comment:
You could use interpolation for this if you want. It's just personal preference.